### DISCUSSION TITLE
Gorilla Web Toolkit discontinued/archived/unmaintained

### DISCUSSION THREAD

**Discussion by einhirn** [dexidp/dex#2933] [Q&A] [answered]
Hi all,

I stumbled across the fact that dex uses the gorilla web toolkit which is discontinued/archived/unmaintained since end of 2022 - are there any plans to switch to another web toolkit or fork and maintain the gorilla web toolkit?

An authentication server using a web toolkit with potential issues that won't get fixed makes me a little queasy, to be honest...

Thanks!

**Comment by sagikazarmark** [comment, marked as answer]
It's not a high priority on our list right now. Although gorilla is discontinued, it's probably the most mature router implementation in the ecosystem, so I'm not worried it's going to become filled with security holes over night.

But you are right that we should migrate to something else. I've been looking at chi router lately as another minimalist solution (basically everything else is bloatware).

I'll move this discussion into an issue to track the work. PRs are welcome. :)

**Reply by nabokihms** [reply]
https://github.com/dexidp/dex/issues/2934
